test/externalTests/common.sh

```diff
@@ -24,6 +24,13 @@ set -e
 
 CURRENT_EVM_VERSION=london
 
+if [[ -f "$2" ]]
+then
+    ARGS=("$@")
+    ARGS[1]="$(realpath "$2")"
+    set -- "${ARGS[@]}"
+fi
+
```

Comment:
I am not sure about this solution. It works, but looks risky and more like a workaround than a proper solution. Can't we just use `realpath` to setup "BINARY_PATH" variable for each test suite (e.g. test/externalTests/gnosis.sh:31) like we do for "REPO_ROOT?

Reply:
I was having those concerns too and was waiting to discuss it further today. Nice that you already reviewed. I think we should go this way you suggest, unless someone else has a better proposal.